packages/react/src/components/Campaign/Proposition.js

```diff
@@ -40,6 +44,8 @@ const ToggleButton = styled.div`
   line-height: 28px;
   padding: 20px 0;
   font-family: ${({ theme }) => theme.headerFontFamily};
+  text-transform: ${({ theme }) =>
+    theme.uppercaseTitles ? 'uppercase' : 'none'};
```

Comment:
Same here.

Reply:
This isn't a crane element.

Reply:
Plus, we already use the theme throughout the React package. If that is something to refactor out I think it is outside of the scope of this PR.